### PR TITLE
[JA] Translate date-time 1.0.2

### PR DESCRIPTION
**Pull request by starfish719**
I translated `lessons/basics/date-time` section.
Please review! @elixirschool/japanese

- bump version

https://github.com/elixirschool/elixirschool/commit/482398d61a27cd3dd26d39a5e68c9222becf9ced#diff-0d7fe8c56d01e42d4fe8ca8b73965a58
https://github.com/elixirschool/elixirschool/commit/8fbad8ea88f4dc692491f3df15c2d95928730f3c#diff-0d7fe8c56d01e42d4fe8ca8b73965a58

### CODE REVIEW

File: ja/lessons/basics/date-time.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ true
 ## NaiveDateTime
 
 Elixirには日付と時間を同時に含む構造体が2種類あります。
-そのうち1つは`NaiveDateTime`です。
+最初に紹介するのは`NaiveDateTime`です。
```
Comment (marocchino) [inline ja/lessons/basics/date-time.md]: (maybe) Is not it needs space around the quotes?
```suggestion
最初に紹介するのは `NaiveDateTime` です。
```

File: ja/lessons/basics/date-time.md [hunk 2]
```diff
@@ -103,5 +103,5 @@ iex> DateTime.from_naive(~N[2016-05-24 13:26:08.003], "Etc/UTC")
 {:ok, #DateTime<2016-05-24 13:26:08.003Z>}
 ```
 
-これがそうです！さらに高度な他の機能を使いたい場合は、 [Time](https://hexdocs.pm/elixir/Time.html) 、 [Date](https://hexdocs.pm/elixir/Date.html) 、 [DateTime](https://hexdocs.pm/elixir/DateTime.html) のドキュメントをさらに確認することを考慮するといいでしょう。
+これがそうです！さらに高度な他の機能を使いたい場合は、 [Time](https://hexdocs.pm/elixir/Time.html) 、 [Date](https://hexdocs.pm/elixir/Date.html) 、 [DateTime](https://hexdocs.pm/elixir/DateTime.html)、 [NaiveDateTime](https://hexdocs.pm/elixir/NaiveDateTime.html) のドキュメントをさらに確認することを考慮するといいでしょう。
```
Comment (marocchino) [inline ja/lessons/basics/date-time.md]: whitespace inconsistency
```suggestion
これがそうです！さらに高度な他の機能を使いたい場合は、 [Time](https://hexdocs.pm/elixir/Time.html) 、 [Date](https://hexdocs.pm/elixir/Date.html) 、 [DateTime](https://hexdocs.pm/elixir/DateTime.html) 、 [NaiveDateTime](https://hexdocs.pm/elixir/NaiveDateTime.html) のドキュメントをさらに確認することを考慮するといいでしょう。
```

**Comment by starfish719** [comment]
@marocchino thank you for your review :)

File: ja/lessons/basics/date-time.md (hunk 1 above)
Comment (marocchino) [reply ja/lessons/basics/date-time.md]: 73, 90 lines still remained.
note: need auto-correction for this later on.

Comment (riseshia) [reply ja/lessons/basics/date-time.md]: oops I missed this 🙈

**Pull request merged**